build_runner/lib/src/entrypoint/base_command.dart

```diff
@@ -97,7 +97,12 @@ abstract class BuildRunnerCommand extends Command<int> {
               'If multiple filters are applied then outputs matching any filter '
               'will be built (they do not need to match all filters).')
       ..addMultiOption(enableExperimentOption,
-          help: 'A list of dart language experiments to enable.');
+          help: 'A list of dart language experiments to enable.')
+      ..addFlag(
```

Comment:
Should we consider defaulting this to on? If we think it is better, then I think it probably makes sense to do? It isn't likely a lot of people will enable it otherwise.

Reply:
I have enabled the option by default if the low-resources option is _not_ enabled. I think it's a better default behavior, but the in-memory overlay used by the delayed writes contradicts the intention of using less memory in low-resources.

Reply:
SGTM